grafana/grafana
# Annotations: Avoid crash when listing panels (that are rows)

Before
The panels list was including rows, which was causing the page to crash.

![rows](https://user-images.githubusercontent.com/88068998/233121876-3edec122-49f6-44f9-bbd1-6d86104a2ec3.png)


After
![no_rows](https://user-images.githubusercontent.com/88068998/233121602-44d10889-466b-43d2-9f11-3459da671ac3.png)


Fixes #66862 


**Testing steps:**

1. In a new/existing dashboard and add some rows
2. Go to dashboard Settings
3. Click on Annotations tab
4. Go to "Show In" section
5. Choose "Selected panels"

Expected results:

- You should only see a list of panels (no rows)
- The page should not crash





Please check that:
